pyproject.toml

```diff
@@ -61,3 +61,14 @@ pytest = "^7.0"
 [build-system]
 requires = ["poetry-core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
+
+[project]
+name = "tomte"
+authors = [
+    {name = "David Minarsch", email = "david.minarsch@googlemail.com"},
+]
+readme = "README.md"
+license = {file = "LICENSE"}
+requires-python = ">=3.7"
+description = "A library that wraps many useful tools (linters, analysers, etc) to keep Python code clean, secure, well-documented and optimised."
+version = "0.2.3"
```

Comment:
I've followed [this](https://flit.pypa.io/en/stable/pyproject_toml.html) and [this](https://peps.python.org/pep-0621/#license). Do we have a way to test if this will be enough?

Reply:
There is now duplication relative to section `tool.poetry`. I think you need to fix that and it indicates to me an issue with this apporach

Reply:
Ok, I've now added the license to `tool.poetry` following their docs. Let's check if this is enough to solve the missing license issue.